conductr_cli/host.py

```diff
@@ -81,10 +82,9 @@ def is_listening(ip_addr, port):
 
 def addr_alias_setup_instructions(addrs, subnet_mask):
     def info_text(commands):
-        if len(commands) > 1:
-            return 'Run the following commands to create the address aliases on your machine:'
-        else:
-            return 'Run the following command to create an address alias on your machine:'
+        return headline(
```

Comment:
I am wondering if we can add the address aliases automatically after the user entered the sudo password. When prompting for the password, we could explain what we are doing. This way we wouldn’t need to highlight the message in a particular fashion.

What do you think?